test/route.js

```diff
@@ -174,11 +174,15 @@ describe('Routing feature server requests', () => {
   })
 
   describe('Query', () => {
+    // This pattern is used to help insulate bugs that occur from platform-specific logic in farmhash.
+    // The existing hard-coded OBJECTID's were generated by a unix-based farmhash, and would cause
+    // the tests to fail when run on windows.
+    var secondOBJECTID
```

Comment:
Same comment as above.  Use `beforeEach` to get objectId data you need for the tests and store on `this.currentTest` as shown [here](https://gist.github.com/icirellik/b9968abcecbb9e88dfb2).